app/(pages)/_components/Textbox/Textbox.tsx

```diff
@@ -0,0 +1,103 @@
+import styles from './Textbox.module.scss';
+import Image from 'next/image';
+import { useState } from 'react';
+
+interface TextboxProps {
+  name: string;
+  placeholder: string;
+}
+
+const Textbox: React.FC<TextboxProps> = (props) => {
```

Comment:
I was originally thinking that we would just use a Rich Text Editor library. If there is one, it might make this a little easier to implement for the future.